app/views/timeline_events/show.html.erb

```diff
@@ -31,6 +31,22 @@
           <% end %>
         </div>
       <% end %>
+      <% @submission.startup_feedback.includes(:faculty).each do |feedback |%>
```

Comment:
`includes` doesn't have the full set of data required to render this page; 

- `feedback.faculty.name` will make `n+1` queries to `User` table for fetching the name; 
- The avatar call will also make `n+1` calls;